packages/fxa-content-server/tests/functional/delete_account.js

```diff
@@ -51,10 +54,41 @@ registerSuite('delete_account', {
               selectors.SETTINGS_DELETE_ACCOUNT.DETAILS
             )
           )
-          .then(fillOutDeleteAccount(PASSWORD))
+          .findAllByCssSelector(selectors.SETTINGS_DELETE_ACCOUNT.CHECKBOXES)
+          .then(checkboxes => checkboxes.map(checkbox => checkbox.click()))
+          .end()
+          .then(
+            type(selectors.SETTINGS_DELETE_ACCOUNT.INPUT_PASSWORD, 'PASSWORD')
+          )
+          .then(click(selectors.SETTINGS_DELETE_ACCOUNT.SUBMIT))
+          .then(
+            visibleByQSA(
+              selectors.SETTINGS_DELETE_ACCOUNT.TOOLTIP_INCORRECT_PASSWORD
+            )
+          )
+      );
+    },
+
+    'sign in, delete account with correct password': function() {
```

Comment:
I think you can combine the two `sign in, delete account`. In this test case, before you use the correct password, you could use the invalid password.

Reply:
I made the suggested change.